module/VuFind/src/VuFind/DigitalContent/OverdriveConnector.php

```diff
@@ -227,6 +227,29 @@ public function getAccess($refresh = false)
         return $result;
     }
 
+    /**
+     * Is Overdrive content active?
+     *
+     * @return bool
+     */
+    public function getContentActive()
+    {
+        $config = $this->getConfig();
+        if ($config->showMyContent == 'always') {
+            return true;
+        } elseif ($config->showMyContent == 'never') {
+            return false;
+        } else {
+            //assume that it is accessOnly
```

Comment:
While refactoring anyway, might as well clean up this minor spacing issue...

```suggestion
            // assume that it is accessOnly
```